### ISSUE TITLE
'auth_header' is an invalid option for 'http' even after installation

### ISSUE THREAD

**Issue by Robin-Sch** [BeryJu/hass-auth-header#328]
I installed hacs and hass-auth-header, however, whenever I try to start HA I get the following error/warnings:

```
WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration auth_header which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
ERROR (MainThread) [homeassistant.config] Invalid config for 'http' at configuration.yaml, line 43: 'auth_header' is an invalid option for 'http', check: http->auth_header, please check the docs at https://www.home-assistant.io/integrations/http
ERROR (MainThread) [homeassistant.setup] Setup failed for 'http': Invalid config.
```

With my config:
```
http:
  use_x_forwarded_for: true
  trusted_proxies: ...
  auth_header:
    debug: true
```

**Comment by Robin-Sch** [comment]
My bad... For some reason I thought auth_header should be indented under http.... Oops

**Comment by Robin-Sch** [comment]
Now I have a different error:

```
http:
  use_x_forwarded_for: true
  trusted_proxies: ...

auth_header:
  username_header: Gap-Auth
  allow_bypass_login: false
  ```
  
  gives
  
  ```
ERROR (MainThread) [homeassistant.config] Invalid config for 'auth_header' at configuration.yaml, line 47: 'allow_bypass_login' is an invalid option for 'auth_header', check: auth_header->allow_bypass_login, please check the docs at https://github.com/BeryJu/hass-auth-header
ERROR (MainThread) [homeassistant.setup] Setup failed for custom integration 'auth_header': Invalid config.
  ```

**Comment by BeryJu** [comment]
@Robin-Sch that option was not present in any released version, I just tagged v1.11 which includes that option
